tsb/fqdn/aws/main.tf

```diff
@@ -0,0 +1,12 @@
+data "terraform_remote_state" "infra" {
+  backend = "local"
+  config = {
+    path = "../../../infra/${var.tsb_mp["cloud"]}/terraform.tfstate.d/${var.tsb_mp["cloud"]}-${var.tsb_mp["cluster_id"]}-${local.k8s_regions[tonumber(var.tsb_mp["cluster_id"])]}/terraform.tfstate"
+  }
+}
```

Comment:
Why is this needed, here and in the other modules, if we don't need to access the cluster info to register the fqdn?

Reply:
I have added it for the sake of consistency so in the future, whoever wants to expand zones similar to gcp will follow the format instead of jsondecode, however you are right, we can take it off here.